pq/pq.go

```diff
@@ -0,0 +1,76 @@
+// Package pq wraps the lib/pq package to instrument all the things.
+package pq
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"net"
+	"time"
+
+	"github.com/lib/pq"
+	"github.com/remind101/pkg/metrics"
+)
+
+func init() {
+	sql.Register("postgresx", &drv{})
```

Comment:
If we're gonna extract this and make it more generic, I would decompose the types in this package, and remove this call, so that apps can register a driver with the stuff they need (e.g. separate out adding metrics, and adding keep alives).